app/templates/admin/vehicles/index.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'admin/base.html' %}
+{% set page_title='Vehicles' %}
+
+{% block action_buttons %}
+  <a href="/admin/vehicles/add" type="button" class="btn btn-sm btn-primary">Add New</a>
+{% endblock %}
+
+{% block content %}
+  <table class="table table-hover table-bordered">
+    <thead>
+      <tr>
+        <th scope="col">id</th>
+        <th scope="col">title</th>
```

Comment:
the title won't be very clear, I guess we can use make/model instead and concatenate the two fields together.

Reply:
lets also add the following fields:
1. year
2. odometer
3. price
4. title
I don't think we will need the description much here,

Reply:
I'm sorry but seems like `odometer` is not in the list of fields?

Reply:
oh I thought I added that as a note to the vehicles model, can you also add the `odometer` to the model

Reply:
This has fixed